webapp/locales/en.json

```
@@ -415,7 +415,7 @@
     "menu": {
       "edit": "Edit Post",
       "delete": "Delete Post",
-      "pin": "Pin post",
+      "pin": "Pin Post",
```

```suggestion
      "pin": "Pin post",
```
we agreed to capitalize the first word in a sentence and lowercase all others....